src/workerd/io/worker-interface.capnp

```diff
@@ -114,6 +114,7 @@ struct QueueMessage @0x944adb18c0352295 {
   id @0 :Text;
   timestampNs @1 :Int64;
   data @2 :Data;
+  contentType @3 :Text;
```

Comment:
Part of me really wants to turn this into an enum instead, but I'll try not to dwell on it :)